docs/format/Hook.md

```diff
@@ -11,6 +11,8 @@ Hooks are defined as part of capacities by maps detailing properties and their a
 ```
 
 Extensions can then specify hooks that will match associated host capacities.
+Hooks implemented by extensions must be qualified with an extra id. This id will be globally unique across all extensions per hook.
```

Comment:
why we can't use extension id?

Reply:
An extension can contain several hooks. You can have several hooks from same type implemented in the same extension (e.g. a basic collectible extension that would implement `status.collectible.CK` and `status.collectible.STRK`; or multiple chat commands)